fudge-ratio/target/fudge_ratio-1/index.html

```diff
@@ -9,10 +9,39 @@
   </head>
```

Comment:
We definitely need to stop the target folder from uploading, probably I will fo it in a next PR